web/client/epics/__tests__/wfsquery-test.js

```diff
@@ -94,6 +94,38 @@ describe('wfsquery Epics', () => {
             }
         });
     });
+    it('wfsQueryEpic passes filterObj', (done) => {
```

Comment:
i would use a more meaningful test title here, aren't you testing that is using default sorting if not provided one?

Reply:
Yes. I will modify the titles accordingly